includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -237,10 +237,10 @@ public function get_fields( $key ) {
 	 * @return int
 	 */
 	protected function sort_by_priority( $a, $b ) {
-		if ( $a['priority'] == $b['priority'] ) {
+		if ( intval( $a['priority'] ) === intval( $b['priority'] ) ) {
```

Comment:
This breaks priority sorting when using decimals (which is very popular), think we should change this back or switch to `floatval` instead of `intval`